client/utils/newGame/newGame.test.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { shallow } from 'enzyme'
```

Comment:
Should consider swapping this out for `react-testing-library`: https://github.com/kentcdodds/react-testing-library

Check out this tweet: https://twitter.com/housecor/status/1115265843917725696

Reply:
I have added it as a new issue so that I can merge this PR